src/hooks/useTransfer.js

```diff
@@ -41,10 +48,27 @@ export const useTransferToL2 = () => {
         const isEthToken = utils.token.isEth(symbol);
         const bridgeContract = getTokenBridgeContract(bridgeAddress);
         const depositHandler = isEthToken ? depositEth : deposit;
+        const tokenContract = isEthToken && getTokenContract(tokenAddress);
         logger.log('Prepared contract and handler', {isEthToken, bridgeContract, depositHandler});
+
+        const isReachedMaxTotalBalance = async () => {
+          const tokenMaxTotalBalance = await maxTotalBalance({decimals, contract: bridgeContract});
+          const tokenBalance = isEthToken
+            ? await ethBalanceOf(bridgeAddress[chainId])
+            : await balanceOf({
+                account: tokenContract,
+                decimals,
+                contract: getL1TokenContract(tokenAddress)
```

Comment:
Why not just `tokenContract`?

Reply:
the function definition asks specifically arguments with names: account, decimals, contract

Reply:
Again:
Why not just `contract: tokenContract`